### PR TITLE
chore(wip): deprecate StatusUtil

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-1239

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

File: src/utilities/StatusUtil/scripts/StatusUtil.js [hunk 1]
```diff
@@ -14,6 +14,9 @@ angular.module('encore.ui.utilities')
  * </pre>
```
Comment (CITguy) [inline src/utilities/StatusUtil/scripts/StatusUtil.js]: Please add a `@deprecated` tag to the ngdocs.

Comment (cathysiller) [inline src/utilities/StatusUtil/scripts/StatusUtil.js]: updated, very top of file?

File: src/utilities/StatusUtil/scripts/StatusUtil.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 angular.module('encore.ui.utilities')
 /**
+ * @deprecated This directive will be removed in EncoreUI 4.0.0
```
Comment (CITguy) [inline src/utilities/StatusUtil/scripts/StatusUtil.js]: `directive` -> `service`

Comment (cathysiller) [inline src/utilities/StatusUtil/scripts/StatusUtil.js]: updated

**Comment by parlarjb** [comment]
Vidyo LGTM

**Pull request merged**